qemu/tests/qemu_img.py

```diff
@@ -498,6 +501,61 @@ def rebase_test(cmd):
         remove(sn2)
         remove(sn1)
 
+    def _amend(cmd, img_name, img_fmt, options):
+        """
+        Simple wrapper of 'qemu-img amend'.
+
+        :param cmd: qemu-img base command
+        :param img_name: image name that should be amended
+        :param img_fmt: image format
+        :param options: a comma separated list of format specific options
+        """
+
+        msg = "Amend '%s' with options '%s'" % (img_name, options)
+        cmd += " amend"
+        if img_fmt:
+            cmd += " -f %s" % img_fmt
+        cache = params.get("cache_mode", '')
+        if cache:
+            cmd += " -t %s" % cache
+        if options:
+            cmd += " -o "
+            for option in options:
+                cmd += "%s=%s," % (option, params.get(option))
+            cmd = cmd.rstrip(',')
+        cmd += " %s" % img_name
+        error.context(msg, logging.info)
+        check_command_output(utils.run(cmd, ignore_status=True))
```

Comment:
This patch is based on #726. The function you mentioned is defined in that patch. @PandaWei